cypress/e2e/ui/Automation/Embedded-Automate/Explorer/class.cy.js

```diff
@@ -29,7 +29,8 @@ describe('Automation > Embedded Automate > Explorer', () => {
     cy.get('[name="description"]').type('This is a test NameSpace');
     cy.get('.bx--btn--primary').contains('Add').click();
 
-    cy.wait(1000); // Need this wait or else namespace doesn't get added properly
+    // Wait for namespace to be visible
+    cy.get('[title="Automate Namespace: TestNameSpace"]', {timeout: 1000}).should('be.visible')
```

Comment:
Doesn't the get imply visibility? (i.e. why do we need the extra check?)

Reply:
Of course, the code could be changed to make it easier to test but I was trying to remove the waits and this was the safest option I could come up with.

Reply:
Ok I thought get only returned when something was visible, but I was mistaken in that in only returns when something is _present_.